db.py

```diff
@@ -69,12 +69,19 @@ def delete_all_restaurants_from_user(self, user_name):
         with self.con:
             self.con.execute("DELETE FROM person_to_restaurant WHERE person_id=(select id from person where name=?)", (user_name,))
 
-    def add_restaurant(self, restaurant):
+    def add_restaurant(self, restaurant_name, restaurant_comment):
```

Comment:
Explicit > Implicit. Optional parameters create bugs, especially when all they'll save is a "" parameter.